src/overlays/actors/ovl_En_Fg/z_en_fg.h

```diff
@@ -5,11 +5,38 @@
 
 struct EnFg;
 
+typedef void (*EnFgActionFunc)(struct EnFg*, GlobalContext*);
+
+typedef struct {
+    /* 0x00 */ AnimationHeader* animationSeg;
+    /* 0x04 */ f32 playbackSpeed;
+    /* 0x08 */ s16 frame;
+    /* 0x0A */ s16 frameCount;
+    /* 0x0C */ u8 mode;
+    /* 0x0E */ s16 transitionRate;
+} EnFgAnimation; // size = 0x10
```

Comment:
https://discord.com/channels/688807550715560050/688851997352263820/843284808992948234
This conversation may be relevant. The `s16`s are different than the typical `f32`s seen in other Animation structs, but then are later casted to `f32`. This struct seems to occur in other actors and may be more of a global struct

Reply:
Are you going to move this to be more of a global header in this PR, or are we going to wait for a later date?

Reply:
I'll leave this for now, but I left a comment about it